cmd/runtime_options_test.go

```diff
@@ -212,39 +206,6 @@ func TestRuntimeOptions(t *testing.T) {
 				Env:                  map[string]string{"test1": "val1"},
 			},
 		},
-		"ignore invalid name system env var name 1": {
-			useSysEnv: false,
-			systemEnv: map[string]string{"test a": "val1", "valid": "val2"},
-			cliFlags:  []string{"--include-system-env-vars=true"},
-			expErr:    false,
-			expRTOpts: lib.RuntimeOptions{
-				IncludeSystemEnvVars: null.NewBool(true, true),
-				CompatibilityMode:    defaultCompatMode,
-				Env:                  map[string]string{"valid": "val2"},
-			},
-		},
```

Comment:
Should we reverse and keep them? In the end, they are still good tests.

Reply:
I belive it's better to do in the task https://github.com/grafana/k6/issues/2730